index.html

```diff
@@ -594,52 +596,57 @@
+We then _define_ the **`subsurface_color`** $\mathbf{C}$ to parametrize the fraction of the energy transmitted through the interface that is transmitted back out and observed, i.e.:
+\begin{equation} \label{subsurface_albedo_constraint}
+\mathbf{E}_\mathrm{multi-scatter} = \left( 1 - \mathbf{E}_\mathrm{spec} \right) \mathbf{C} \ .
 \end{equation}
-The physical RGB scattering coefficient is then given by $\boldsymbol{\mu}_s = \boldsymbol{\alpha}\,\boldsymbol{\mu}_t$.
+Thus since $0 \le \mathbf{C} \le 1$, $\mathbf{E}_\mathrm{subsurface} \le 1$ so energy is always conserved, and if $\mathbf{C}=1$ then $\mathbf{E}_\mathrm{subsurface}=1$ which guarantees that a white **`subsurface_color`** passes the furnace test. According to this definition, the **`subsurface_color`** $\mathbf{C}$ is the observed reflection color (viewed at normal incidence under uniform illumination) in areas where the Fresnel reflection is negligible, and otherwise the observed color is a blend of $\mathbf{C}$ with the gray Fresnel reflection that conserves total reflected energy.
 
-This relationship was derived (via fitting) in the isotropic $g=0$ case. To account for anisotropy, similarity theory [#d'Eon2021] shows that a medium with anisotropy $g$ will have approximately similar overall light transport to an isotropic medium (while still exhibiting noticeably different directional effects, especially in low-order scattering) provided that the scattering coefficient is remapped to $\bar{\boldsymbol{\mu}}_s = \boldsymbol{\mu}_s / (1 - g)$, while keeping the absorption coefficient the same. Applying this, the net result is that the remapped single-scattering albedo $\bar{\boldsymbol{\alpha}}$ is given by
+Given the required multiple-scattering albedo $\mathbf{E}_\mathrm{multi-scatter}$ according to the formula above, then in principle, the single-scattering albedo $\boldsymbol{\alpha}$ which generates the required $\mathbf{E}_\mathrm{multi-scatter}$ can be determined. We do not require any particular theoretical formula for $\boldsymbol{\alpha}(\mathbf{C})$ be used, but the closer it is to satisfying equation [subsurface_albedo_constraint] the closer it is to the ground truth. Ideally, this formula should take into account all the properties of the medium as well as the dielectric interface. A number of such approximate formulas have been derived in the literature. A well-known approximation is due to van de Hulst ([#Kulla2017], [#d'Eon2021]), which assumes an index-matched boundary (i.e. $\mathbf{E}_\mathrm{spec} = 0$, thus $\mathbf{E}_\mathrm{multi-scatter} = \mathbf{C}$), where:
```

Comment:
> all the properties of the medium as well as the dielectric interface

I think this part could potentially be misinterpreted to include the shape of the object, however clarifying the semi-infinite homogeneous slab assumption mentioned above could resolve this.

Reply:
Hopefully the re-wording mentioned in https://github.com/AcademySoftwareFoundation/OpenPBR/pull/171#discussion_r1571165046 addresses this.